.github/workflows/dart_ci.yml

```diff
@@ -38,8 +38,12 @@ jobs:
         if: always() && ${{ matrix.sdk }} == 'stable' && steps.install.outcome == 'success'
 
       # Analyze before generated files are created to verify that component boilerplate analysis is "clean" without the need for building
-      - name: Analyze project source (pre-build)
-        run: pub run dart_dev analyze
+      - name: Analyze example source (pre-build)
+        run: |
+          cd example/boilerplate_versions
+          if [ ${{ matrix.sdk }} = 'stable@2.7.2' ]; then pub global activate tuneup && tuneup check; fi
```

Comment:
@sydneyjodon-wk since we're [using the new "official" setup-dart GH action](https://github.com/Workiva/over_react/pull/673), you'll want to merge in the latest from the `dart-2.9-boilerplate-wip` branch, and then replace `stable@2.7.2` with just `2.7.2` anywhere you've used it in this PR.

Reply:
Updated in 5f5a492